src/ctia/flows/hooks/event_hooks.clj

```diff
@@ -72,11 +93,13 @@
     :nothing)
   (handle [_ event _]
     (when (judgement? event)
-      (some-> (:judgement store/stores)
-              deref
-              (store/calculate-verdict event)
-              ;; TODO: store in verdict store. Issue #277
-              ))
+      (let [{{observable :observable :as judgement} :entity owner :owner} event]
+        (when-let [new-verdict (some->
+                                (store/read-store :judgement
+                                                  store/calculate-verdict
+                                                  observable)
+                                (realize-verdict-wrapper judgement owner))]
+          (store/write-store :verdict store/create-verdict new-verdict))))
```

Comment:
Once #355 is done, this needs to change to using the CRUD flow, instead of writing directly to the store, so verdict creation can be hooked